src/garage/trainer/trainer.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, total_epoch, total_itr, total_env_steps, last_episode):
 
 class SetupArgs:
     # pylint: disable=too-few-public-methods
-    """Arguments to setup a runner.
+    """Arguments to setup a trainer.
```

Comment:
unless we feel that we are going to have a lot of these, perhaps we can just put this in `src/garage/trainer.py` (and also `TFTrainer` there too)